.gitlab-ci.yml

```diff
@@ -22,6 +22,10 @@ stages:
   - dockerize
   - deploy
 
+.on-release-prod:                   &on-release-prod
+  only:
+    - /^v[0-9]+\.[0-9]+.*$/
```

Comment:
`$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+.*$/` would be more correct

Reply:
will fix 👍  for the sake of learning - could you explain why?
fyi: i copied from cla bot,

Reply:
With the way it's implemented currently, this expression would target both branches and tags. `$CI_COMMIT_TAG` is only for tags specifically, which I infer is your intention.